incubator/hnc/pkg/controllers/hierarchy_controller.go

```diff
@@ -545,6 +546,14 @@ func (r *HierarchyReconciler) getNamespace(ctx context.Context, nm string) (*cor
 	return ns, nil
 }
 
+func (r *HierarchyReconciler) updateReconcileCounter(log logr.Logger, ns string) {
+	// Only count number of hier reconciles on user-created namespaces.
+	if !isExcludedNS(ns) {
+		hierTotal <- 1
+		log.Info("hierTotalReconciles increment")
```

Comment:
Can we take out the log message? Then we also wouldn't have to pass in the log object.